sources/packages/backend/apps/api/src/route-controllers/application/application.controller.service.ts

```diff
@@ -168,14 +169,25 @@ export class ApplicationControllerService {
    */
   async transformToApplicationDTO(
     application: Application,
-  ): Promise<ApplicationBaseAPIOutDTO> {
+  ): Promise<ApplicationSupplementalDataAPIOutDTO> {
     return {
       data: application.data,
       id: application.id,
       applicationStatus: application.applicationStatus,
       applicationNumber: application.applicationNumber,
       applicationFormName: application.programYear.formName,
       applicationProgramYearID: application.programYear.id,
+      studentFullName: getUserFullName(application.student.user),
+      applicationOfferingIntensity:
+        application.currentAssessment.offering?.offeringIntensity,
+      applicationStartDate: getDateOnlyFormat(
```

Comment:
`getDateOnlyFormat` is not needed for Date-only dates.
Only code may still use it, but it should not be needed.

Reply:
Moved where the date is being formatted.